libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -30,50 +33,76 @@ void VotePacketHandler::process(const PacketData &packet_data, const std::shared
     const auto vote_hash = vote->getHash();
     LOG(log_dg_) << "Received PBFT vote " << vote_hash;
 
+    // Check this before we are marking vote as seen to have an opportunity to process it after
+    if (vote->getRound() > pbft_mgr_->getPbftRound() + kVoteAcceptingRounds) {
+      LOG(log_nf_) << "Skip vote " << vote_hash.abridged() << ". Round too big " << pbft_mgr_->getPbftRound() << " "
+                   << vote->getRound();
+      // Do not request round sync too often here
+      if (std::chrono::system_clock::now() - round_sync_request_time_ > kSyncRequestInterval) {
+        // request sync from this node
+        sealAndSend(peer->getId(), GetVotesSyncPacket, std::move(dev::RLPStream(2) << pbft_mgr_->getPbftRound() << 0));
```

Comment:
I guess this requesting mechanism cannot work anymore without providing also period...